install/gcp/gcp_installer.py

```diff
@@ -87,6 +87,21 @@ def run():
     client_config = ClientConfig(**args)
     server_config = ServerConfig(**args)
 
+    deprecation_message = (
+        '\n\n'
+        '##################################################################\n'
+        '#  ATTENTION: This python installer will be deprecated           #\n'
+        '#  on 9/30/2019, and will be replaced by the easier and simpler  #\n'
+        '#  to use terraform installer.  Try out this short, interactive  #\n'
+        '#  tutorial to learn how to use the terraform installer.         #\n'
+        '#  http://foobar.com                                             #\n'
```

Comment:
Can we just link to https://forsetisecurity.org/docs/latest/setup/install.html ?

Reply:
Done